data/common.yaml

```diff
@@ -1,36 +1,36 @@
 ---
-# Default Installation parameters
-puppet_metrics_dashboard::add_dashboard_examples: false
 puppet_metrics_dashboard::manage_repos: true
+
+puppet_metrics_dashboard::add_dashboard_examples: false
 puppet_metrics_dashboard::overwrite_dashboards: true
-puppet_metrics_dashboard::use_dashboard_ssl: false
-puppet_metrics_dashboard::dashboard_cert_file: "/etc/grafana/%{clientcert}_cert.pem"
-puppet_metrics_dashboard::dashboard_cert_key: "/etc/grafana/%{clientcert}_key.pem"
-puppet_metrics_dashboard::influxdb_database_name:
-  - "telegraf"
-puppet_metrics_dashboard::grafana_version: "5.1.4"
-puppet_metrics_dashboard::grafana_http_port: 3000
-puppet_metrics_dashboard::influx_db_password: "puppet"
-puppet_metrics_dashboard::grafana_password: "admin"
-puppet_metrics_dashboard::consume_graphite: false
-# Influxdb TICK stack
-puppet_metrics_dashboard::enable_telegraf: true
-puppet_metrics_dashboard::enable_kapacitor: false
+
 puppet_metrics_dashboard::enable_chronograf: false
-# telegraf config
+puppet_metrics_dashboard::enable_kapacitor: false
+puppet_metrics_dashboard::enable_telegraf: true
 puppet_metrics_dashboard::configure_telegraf: true
-puppet_metrics_dashboard::master_list:
-  - "%{trusted.certname}"
-puppet_metrics_dashboard::puppetdb_list:
-  - "%{trusted.certname}"
-puppet_metrics_dashboard::postgres_host_list:
-  - "%{trusted.certname}"
```

Comment:
The above are the only deletions. The remainder of the differences are grouping and sorting.

Reply:
The `localhost_or_hosts_with_pe_profile` function called as the default value for the above three parameters will return `$trusted['certname']` if the query fails, resulting in the same defaults.